frontend/components/modals/LockedModal.tsx

```diff
@@ -188,6 +191,24 @@ function LockedModalButton({
   const unvestedPreUnlockAll = useUnvestedPreUnlockAllMutation()
   const unvestedUnlockAll = useUnvestedUnlockAllMutation()
 
+  const [isLlcModalOpen, setIsLlcModalOpen] = useState(false)
+
+  const onStakeAll = useCallback(async () => {
+    if (mainStakeAccount === 'NA' || mainStakeAccount === undefined) return
+    try {
+      const isLlcMember = await stakeConnection!.isLlcMember(mainStakeAccount)
+
+      if (isLlcMember === true)
+        unvestedLockAll.mutate({
+          mainStakeAccount: mainStakeAccount,
+          stakeConnection: stakeConnection!,
+        })
+      else setIsLlcModalOpen(true)
+    } catch {
+      toast.error('Error: staking')
```

Comment:
This error feels weird

Reply:
stakeConnection.isLlcMember can fail, right?
I can change the message or should i remove it?